pkgs/development/python-modules/karney/default.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hypothesis,
+  numpy,
+  poetry-core,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "karney";
+  version = "1.0.10";
+  pyproject = true;
+
+  doCheck = true;
+
+  src = fetchFromGitHub {
+    inherit version;
+    owner = "pbrod";
+    repo = "karney";
+    tag = "v${version}";
+    hash = "sha256-UuHj7fBtWH6+kFsrOQya9odeQ1hR4UdlyXK48G4gH7o=";
+  };
+
+  build-system = [ poetry-core ];
+
+  dependencies = [ numpy ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    hypothesis
+  ];
+
+  pythonImportsCheck = [ "karney" ];
+
+  meta = with lib; {
+    changelog = "https://github.com/pbrod/karney/blob/main/CHANGELOG.md";
```

```suggestion
    changelog = "https://github.com/pbrod/karney/blob/v${version}/CHANGELOG.md";
```
Good try. Not quite.